data-es6.js

```diff
@@ -5796,6 +5801,26 @@ exports.tests = [
         res: methods[m]
       }
     };
+    obj['Math.hypot'] = {
+      exec: function(){/*
+        return Math.hypot(1) === 1 && Math.hypot(9, 12, 20) === 25
```

Comment:
Perhaps let's add 0-argument case as well then? (so that we cover 0, 1, 3 (and 2 as part of last one) )

```js
Math.hypot() === 0 && 
Math.hypot(1) === 1 &&
Math.hypot(9, 12, 20) === 25
```